test_src/sljitTest.c

```diff
@@ -9549,7 +9818,6 @@ int sljit_test(int argc, char* argv[])
 	test79();
 	test80();
 	test81();
-
```

Comment:
Please keep this newline.

Reply:
Please revert this change

Reply:
I'm sorry. I forgot about that one. Should be there now.